radix-engine/src/blueprints/resource/auth_zone_composition.rs

```diff
@@ -188,12 +188,13 @@ fn max_ids_locked<Y: KernelSubstateApi + ClientApi<RuntimeError>>(
 ) -> Result<
     (
         BTreeSet<NonFungibleLocalId>,
-        HashMap<LocalRef, BTreeSet<NonFungibleLocalId>>,
+        NonIterMap<LocalRef, BTreeSet<NonFungibleLocalId>>,
```

Comment:
This is an interesting one. I would expect we have a dependency over the iteration order, but it actually just sort the total locked non-fungible local ids.